app/views/comments/_create.html.haml

```diff
@@ -0,0 +1,7 @@
+- @bar.comments.each do |comment|
+  %p= comment.body
+- if user_signed_in?
+  = form_with(model: [ @bar, @bar.comments.build ], local: true) do |form|
```

Comment:
No need for empty spaces after `[` and before `]`. Also please avoid hash rocker in line below and use same style for quotations across the app (double quotes). 
```
 = form.text_area :body, required: true, class: 'form-control'
```